### PR TITLE
Remove unused fixtures

### PR DESCRIPTION
**Pull request by seallard**
## Description
Remove unused fixtures found with `pytest --dead-fixtures`

### Fixed
- Remove unused fixtures

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Review by diitaz93 — APPROVED** [review]
Looks good on my side 👍 I would still wait for the approval of @ChrOertlin and @sofstam before merging as they may be using some of these fixtures in their open PRs.

**Comment by seallard** [comment]
> Looks good on my side 👍 I would still wait for the approval of @ChrOertlin and @sofstam before merging as they may be using some of these fixtures in their open PRs.

Hm, sure. I think its fine though, worst case the fixtures can retrieved from the diff in this commit for example

> Looks good on my side 👍 I would still wait for the approval of @ChrOertlin and @sofstam before merging as they may be using some of these fixtures in their open PRs.

From what I can see, they don't use any of these. Worst case the fixtures can be retrieved from any previous version of the repo

**Pull request merged**
